### PR TITLE
MPlayer cheat sheet created

### PR DESCRIPTION
**Pull request by arwk**
**What does your Instant Answer do?**
MPlayer cheat sheet

**What problem does your Instant Answer solve (Why is it better than organic links)?**
Brief overview of  MPlayer key sequences

**What is the data source for your Instant Answer? (Provide a link if possible)**
http://www.mplayerhq.hu/DOCS/man/en/mplayer.1.html

**Why did you choose this data source?**
Complete key sequence documentation

**Are there any other alternative (better) data sources?**
alternative: http://sheet.shiar.nl/mplayer, but some keys are interchanged

**What are some example queries that trigger this Instant Answer?**
mplayer cheat sheet

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
MPlayer users

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**
No

**Which existing Instant Answers will this one supercede/overlap with?**
None

**Are you having any problems? Do you need our help with anything?**
No

**Where did you hear about DuckDuckHack? (For first time contributors)**
https://duckduckgo.com/

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
mplayer-duckduck-screenshot

-----
IA Page: https://duck.co/ia/view/mplayer_cheat_sheet

### CODE REVIEW

**Comment by moollaza** [comment]
@arwk thanks a lot for contributing, we really appreciate it! We'll be sure to review this shortly and give you some feedback!

File: share/goodie/cheat_sheets/json/mplayer.json [hunk 1]
```diff
@@ -0,0 +1,137 @@
+{
+    "id": "mplayer_cheat_sheet",
+    "name": "Mplayer",
+    "description": "Video Player",
+    "metadata": {
+        "sourceName": "Mplayer manpage",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/mplayer.json]: Let's change this to `MPlayer HQ` so it describes the website

File: share/goodie/cheat_sheets/json/mplayer.json [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+            {
+                "val": "toggle stay-on-top",
+                "key": "T"
+            },
+            {
+                "val": "toggle frame dropping states",
+                "key": "d"
+            },
+            {
+                "val": "adjust contrast",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/mplayer.json]: Do these numbers increase/decrease the contrast/brightness etc? If so I think we should separate these into individual keys:

```
"val": "Increase brightness",
"key": "3"
```

File: share/goodie/cheat_sheets/json/mplayer.json [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+                "val": "seek forward / backward 10min",
+                "key": "[Page up] / [Page down]"
+            },
+            {
+                "val": "halve / double current playback speed",
+                "key": "[\\{] / [\\}]"
+            },
+            {
+                "val": "reset playback speed",
+                "key": "[Backspace]"
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/mplayer.json]: Please remove the `[]` here, they're not needed unless there's spaces in the key

File: share/goodie/cheat_sheets/json/mplayer.json [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+                "val": "step forward / backward in the subtitle list",
+                "key": "[y] / [g]"
+            },
+            {
+                "val": "subtitle delay +/-0.1s",
+                "key": "[x] / [z]"
+            },
+            {
+                "val": "toggle subtitle alignment",
+                "key": "[a]"
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/mplayer.json]: You can remove the `[]` here as well

**Comment by moollaza** [comment]
@arwk thanks a lot for contributing, we really appreciate it! We've left you some feedback that needs to be implemented. Once things are updated please let us know by leaving a comment, so we can review this again. If you have any questions, feel free to ask them here!

**Comment by arwk** [comment]
Thanks for your quick feedback!
I updated the cheat sheet like you advised

**Comment by moollaza** [comment]
@arwk this LGTM :+1:

Thanks a lot for contributing (we really appreciate it :smiley:). We invite you to stick around and comment/help on other PR's & Issues or even submit an Instant Answer!

**Comment by arwk** [comment]
@moollaza I'm just curious about how long it usually takes until the cheet sheet appears on the official DuckDuckGo website

**Comment by moollaza** [comment]
> I'm just curious about how long it usually takes until the cheet sheet appears on the official DuckDuckGo website

Cheat Sheets are usually live in a few days because they have a quicker release process. Others take a few days to a week depending on how soon we can do a release. I'm aiming to get them done weekly depending on how busy I am/we are :+1:

**Comment by moollaza** [comment]
We announce them once they're live and I try to go back to all PR's to notify developers. We're working to automate the PR notifications though seeing as the PR volume is increasing.

**Comment by arwk** [comment]
Is it planned to implement a possibility to translate CheetSheats?
It would be great to translate CheetSheats and display the appropriate language according to the selected country on DuckDuckGo!

**Comment by moollaza** [comment]
@arwk congrats! It's live: https://duckduckgo.com/?q=mplayer+cheat+sheet&ia=cheatsheet

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Comment by moollaza** [comment]
>Is it planned to implement a possibility to translate CheetSheats?

We don't have any immediate plans on our radar to translate the cheat sheet content, but it's something we're considering.

**Pull request merged**